substrate/runtime/executive/src/lib.rs

```diff
@@ -290,7 +290,7 @@ mod tests {
 		});
 	}
 
-	fn new_test_ext() -> runtime_io::TestExternalities<KeccakHasher> {
+	fn new_test_ext() -> runtime_io::TestExternalities<Blake2Hasher> {
```

Comment:
Below, in the `block_import_works`, there are comment that blake state root should be `02532989c613369596025dfcfc821339fc9861987003924913a5a1382f87034a`. I wonder why it isn't changed then? Also, can you update the comment there?